packages/classic/scss/menu/_variables.scss

```diff
@@ -2,51 +2,148 @@
 
 $kendo-menu-padding-x: k-spacing(0) !default;
```

Comment:
Why are we not documenting these variables? Are they unused, if so maybe they can be removed?

Reply:
Yes, they were unused. Will be removed.